src/containers/Auth/Auth.js

```diff
@@ -14,26 +19,83 @@ import classes from './Auth.module.css';
 // }));
 
 class Auth extends Component{
+    state = {
+        isSignUp: false,
+        email: null,
+        password: null
+    }
+
+    emailChangeHandler = (event) => {
+        this.setState({email: event.target.value});
+    }
+
+    passwordChangeHandler = (event) => {
+        this.setState({password: event.target.value});
+    }
+
```

Comment:
This can be combiened in one function called 
inputFieldChangeHandler=()=>{
this.setState({[event.target.name]: event.target.value});
}

Where the name is a passed property to the field